src/pe/header.rs

```diff
@@ -531,11 +769,23 @@ impl CoffHeader {
     }
 }
 
+/// The PE header.
+///
+/// ## Position in a modern PE file
+///
+/// The PE header is located at the very beginning of the file and
+/// is followed by the section table and sections.
 #[derive(Debug, PartialEq, Copy, Clone, Default)]
 pub struct Header {
     pub dos_header: DosHeader,
     /// DOS program for legacy loaders
     pub dos_stub: DosStub,
+
+    // Q (JohnScience): should we care about the "rich header"?
+    // https://0xrick.github.io/win-internals/pe3/#rich-header
```

Comment:
this might be a good thing if it is real and people use it for stuff :shrug:

Reply:
@JohnScience would you mind filing an issue for the rich header?

Reply:
@m4b It is mostly useful for cybersecurity.

Reply:
> @JohnScience would you mind filing an issue for the rich header?

Done, https://github.com/m4b/goblin/issues/400